src/domain/synthetics/accountStats/useIsLargeAccountData.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from "react";
+import { USD_DECIMALS } from "config/factors";
+import { expandDecimals } from "lib/numbers";
+
+import { useAccountVolumeStats } from "./useAccountVolumeStats";
+
+// Thresholds to recognise large accounts
+const MAX_DAILY_VOLUME = expandDecimals(220_000n, USD_DECIMALS);
```

Comment:
sound like constants that should be places in config/factors

Reply:
I'm not sure. This is very specific constants that are only related to this feature. I think it's better to keep them with their logic